### PR TITLE
Lesson 1 4

### PR DESCRIPTION
**Pull request by AE-BHH**
Assignment for 4, lesson_1_4

### CODE REVIEW

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -1,20 +1,31 @@
 import React from 'react'
 
-const AddTodoForm = (props) => {
+function AddTodoForm(props) {
+	const {onAddTodo} = props;
```
Comment (JoHeyJo) [inline src/AddTodoForm.js:4]: This is one of destructuring props. What's another way to do this and eliminate this extra line of code?

File: src/TodoList.js [hunk 2]
```diff
@@ -1,28 +1,12 @@
...
-
-const TodoList = () => {
+function TodoList({ todoList }) {
 	return (
 		<>
 			<ul>
-				{todoList.map((item) => {
-					return <TodoListItem key={item.id}
-					props={item} />
+				{todoList.map((todo) => {
```
Comment (JoHeyJo) [inline src/TodoList.js:8]: Nice renaming of variable!

**Comment by JoHeyJo** [comment]
Nicely done! Fix that one comment and merge it!

**Review by JoHeyJo — APPROVED** [review]

File: src/AddTodoForm.js (hunk 1 above)
Comment (AE-BHH) [reply src/AddTodoForm.js:4]: The other way of destructing will be like this:
const AddTodoForm({props} =>{
......
}

**Comment by JoHeyJo** [comment]
@AE-BHH you can go ahead and merge this branch.

**Pull request merged**